### ISSUE TITLE
Non existing source maps files break the build

### ISSUE THREAD

**Issue by swernerx** [jspm/jspm-cli#1880]
I use the library yahoo/intl-relativeformat. The current version of this library...

```
Unhandled rejection Error on fetch for npm:intl-relativeformat@1.3.0/lib/main.js at file:///Users/Sebastian/Workspace/modern-spa-boilerplate/src/jspm_packages/npm/intl-relativeformat@1.3.0/lib/main.js
	Loading npm:intl-relativeformat@1.3.0/index.js
	Loading app/main.js
	Error: ENOENT: no such file or directory, open '/Users/Sebastian/Workspace/modern-spa-boilerplate/src/jspm_packages/npm/intl-relativeformat@1.3.0/lib/main.js.map'
    at Error (native)
```

The map file which is mentioned is not actually part of the current version of intl-relativformat. So that's indeed an error case (see also https://github.com/yahoo/intl-relativeformat/issues/43). But as source maps are pretty much more optional it should maybe not result into an hard-error but just a warning.

**Comment by swernerx** [comment]
```
jspm --version
0.17.0-beta.17
Running against local jspm install.
```

**Comment by guybedford** [comment]
Thanks, was waiting on this for some real world data, and that makes complete sense... added https://github.com/systemjs/builder/commit/804e63ef036cc7943d9dd07de0876ec1c77bb4e3

**Comment by arackaf** [comment]
@guybedford so sorry to reach out to you both here and on Twitter - not sure how often you check the latter.  Is there any way this builder fix could make its way onto a new version on npm??

**Comment by arackaf** [comment]
I just now see the `pending release` label - sorry, I missed that before.  Sorry if there's a reason it hasn't been released yet.

**Comment by guybedford** [comment]
Released in 0.17.0-beta.18.

**Comment by quentind** [comment]
I am still having issues with missing source maps using jspm `0.17.0-beta.18`. 

The issue stems from the same package as @swernerx: `intl-relativeformat@1.3.0` and from other `intl-*` packages.

Getting this error when trying to build with source maps enabled:

```
{ [Error: Error on fetch for npm:intl-messageformat@1.3.0/lib/main.js at file:///Users/quentind/project/jspm_packages/npm/intl-messageformat@1.3.0/lib/main.js
	Loading npm:intl-messageformat@1.3.0/index.js
	Loading npm:react-intl@2.1.2/lib/index.js
	Loading src/app.js
	Loader fetch hook did not return a source string]
  originalErr: [TypeError: Loader fetch hook did not return a source string] }
```

**Comment by guybedford** [comment]
Corrected the fix in https://github.com/systemjs/builder/commit/a0cc13031b64843b881cd3259a19478aca1e6e80. Apparently I need a holiday.

**Comment by guybedford** [comment]
Released in 0.17.0-beta.19.
